packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -111,26 +113,16 @@ Obsoletes: %{?scl_prefix}rubygem-bastion
 %{?scl:Obsoletes: ruby193-rubygem-%{gem_name}}
 
 # start package.json devDependencies BuildRequires
-BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
-BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 4.2.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 4.6.0
 BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 5.0.0
-BuildRequires: %{?scl_prefix}npm(babel-loader) >= 8.0.0
-BuildRequires: %{?scl_prefix}npm(babel-loader) < 9.0.0
-BuildRequires: %{?scl_prefix}npm(identity-obj-proxy) >= 3.0.0
-BuildRequires: %{?scl_prefix}npm(identity-obj-proxy) < 4.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/find-foreman) >= 4.8.0
```

Comment:
what's that?

Reply:
The result of https://community.theforeman.org/t/rfc-dont-duplicate-npm-packages-in-katello-and-plugins-for-react-testing/18725. The difference is that with Ruby everything is executed from Foreman. Foreman knows where the plugins are and can chdir if needed. The frontend people like to treat plugins as standalone apps and Foreman as a dependency so they come up with their own solutions.

Reply:
k